fast-af/fast-checkout-woocommerce
# PINT-639 - Update for WC Dynamic Pricing & Discounts plugin

# Description

Add an update to support the [WooCommerce Dynamic Pricing & Discounts](https://codecanyon.net/item/woocommerce-dynamic-pricing-discounts/7119279?s_rank=8) plugin. This update should unblock Jaeger Sports.

# Testing instructions

1. Login to [staging](https://dev-fast-wc.pantheonsite.io/).
2. Purchase 5 [hoodies](https://dev-fast-wc.pantheonsite.io/product/hoodie/).
3. Verify that the price per product drops to $42 from $45 in the Fast Checkout.
4. Purchase 10 [hoodies](https://dev-fast-wc.pantheonsite.io/product/hoodie/).
5. Verify that the price per product drops to $40 from $45 in the Fast Checkout.

# Checks
